### ISSUE TITLE
User deletion on user details page creates an error

### ISSUE THREAD

**Issue by mmnosek** [StarcounterPrefabs/UserAdmin#37]
### Steps to reproduce
1. Run Launcher, SignIn, UserAdmin
2. Sign in as Admin
3. Go to System Users
4. Create some user
5. **Go back to created user's page**
6. Click _Delete_
### Expected result

I'm redirected to user list and deleted user is no longer on the list
### Actual result

Error occurs. User is deleted, but I need to refresh a page manually because of an error
### Technical information

DB error log:

```
Starcounter.DbException: ScErrInvalidObjectAccess (SCERR4174): The operation failed because of an invalid object access.
Version: 2.3.0.4048.
Help page: https://github.com/Starcounter/Starcounter/wiki/SCERR4174.
   at Starcounter.Internal.DbState.ReadString(UInt64 oid, UInt64 address, Int32 index) in C:\TeamCity\TeamCity10\buildAgent\work\sc-pnext-nightly-275\Level1\src\Starcounter\Internal\DbState.cs:line 683
   at lambda_method(Closure , Json )
   at Starcounter.Templates.Property`1.CheckAndSetBoundValue(Json parent, Boolean addToChangeLog) in C:\TeamCity\TeamCity10\buildAgent\work\sc-pnext-nightly-275\Level1\src\Starcounter.XSON\Templates\Foundation\Property.cs:line 208
   at Starcounter.Json.<LogObjectValueChangesWithDatabase>b__124_0(ChangeLog clog, Json json, Boolean css) in C:\TeamCity\TeamCity10\buildAgent\work\sc-pnext-nightly-275\Level1\src\Starcounter.XSON\Class Json\Json.Object.ChangeLog.cs:line 311
   at Starcounter.Internal.TransactionManager.Scope[T1,T2,T3](TransactionHandle handle, Action`3 action, T1 arg1, T2 arg2, T3 arg3) in C:\TeamCity\TeamCity10\buildAgent\work\sc-pnext-nightly-275\Level1\src\Starcounter\TransactionManager.cs:line 500
   at Starcounter.Advanced.XSON.JsonExtension.Scope[T1,T2,T3](Json json, Action`3 action, T1 arg1, T2 arg2, T3 arg3) in C:\TeamCity\TeamCity10\buildAgent\work\sc-pnext-nightly-275\Level1\src\Starcounter.XSON\Class Json\JsonExtension.cs:line 139
   at Starcounter.Json.<LogObjectValueChangesWithDatabase>b__124_0(ChangeLog clog, Json json, Boolean css) in C:\TeamCity\TeamCity10\buildAgent\work\sc-pnext-nightly-275\Level1\src\Starcounter.XSON\Class Json\Json.Object.ChangeLog.cs:line 306
   at Starcounter.Advanced.XSON.JsonExtension.Scope[T1,T2,T3](Json json, Action`3 action, T1 arg1, T2 arg2, T3 arg3) in C:\TeamCity\TeamCity10\buildAgent\work\sc-pnext-nightly-275\Level1\src\Starcounter.XSON\Class Json\JsonExtension.cs:line 141
   at Starcounter.Json.<LogObjectValueChangesWithDatabase>b__124_0(ChangeLog clog, Json json, Boolean css) in C:\TeamCity\TeamCity10\buildAgent\work\sc-pnext-nightly-275\Level1\src\Starcounter.XSON\Class Json\Json.Object.ChangeLog.cs:line 368
   at Starcounter.Advanced.XSON.JsonExtension.Scope[T1,T2,T3](Json json, Action`3 action, T1 arg1, T2 arg2, T3 arg3) in C:\TeamCity\TeamCity10\buildAgent\work\sc-pnext-nightly-275\Level1\src\Starcounter.XSON\Class Json\JsonExtension.cs:line 141
   at Starcounter.Json.LogArrayChangesWithDatabase(ChangeLog changeLog, Boolean callStepSiblings) in C:\TeamCity\TeamCity10\buildAgent\work\sc-pnext-nightly-275\Level1\src\Starcounter.XSON\Class Json\Json.Object.ChangeLog.cs:line 245
   at Starcounter.Json.<LogObjectValueChangesWithDatabase>b__124_0(ChangeLog clog, Json json, Boolean css) in C:\TeamCity\TeamCity10\buildAgent\work\sc-pnext-nightly-275\Level1\src\Starcounter.XSON\Class Json\Json.Object.ChangeLog.cs:line 306
   at Starcounter.Advanced.XSON.JsonExtension.Scope[T1,T2,T3](Json json, Action`3 action, T1 arg1, T2 arg2, T3 arg3) in C:\TeamCity\TeamCity10\buildAgent\work\sc-pnext-nightly-275\Level1\src\Starcounter.XSON\Class Json\JsonExtension.cs:line 141
   at Starcounter.Json.LogObjectValueChangesWithDatabase(ChangeLog changeLog, Boolean callStepSiblings) in C:\TeamCity\TeamCity10\buildAgent\work\sc-pnext-nightly-275\Level1\src\Starcounter.XSON\Class Json\Json.Object.ChangeLog.cs:line 380
   at Starcounter.XSON.ChangeLog.Generate(Boolean flushLog, Change[]& changeArr) in C:\TeamCity\TeamCity10\buildAgent\work\sc-pnext-nightly-275\Level1\src\Starcounter.XSON\Sessions\JSON-Patch\ChangeLog.cs:line 96
   at Starcounter.XSON.JsonPatch.Generate(ChangeLog changeLog, Boolean includeNamespace, Boolean flushLog, Byte[]& patches) in C:\TeamCity\TeamCity10\buildAgent\work\sc-pnext-nightly-275\Level1\src\Starcounter.XSON\Sessions\JSON-Patch\JsonPatch.cs:line 115
   at Starcounter.XSON.JsonPatch.Generate(Json json, Boolean flushLog, Boolean includeNamespace, Byte[]& patches) in C:\TeamCity\TeamCity10\buildAgent\work\sc-pnext-nightly-275\Level1\src\Starcounter.XSON\Sessions\JSON-Patch\JsonPatch.cs:line 92
   at Starcounter.Internal.PuppetRestHandler.HandleWebSocketJson(Byte[] bs, WebSocket ws) in C:\TeamCity\TeamCity10\buildAgent\work\sc-pnext-nightly-275\Level1\src\Starcounter.Apps.JsonPatch\BuiltInRestHandlers\PuppetRestHandler.cs:line 64
   at Starcounter.Internal.AllWsGroups.RunHandler(UInt16 managedHandlerId, UInt32 groupId, WebSocket ws) in C:\TeamCity\TeamCity10\buildAgent\work\sc-pnext-nightly-275\Level1\src\Starcounter.Internal\Http\WsManager.cs:line 289
   at Starcounter.GatewayHandlers.HandleWebSocket(UInt16 managedHandlerId, Byte* rawChunk, BMX_TASK_INFO* taskInfo, Boolean* isHandled) in C:\TeamCity\TeamCity10\buildAgent\work\sc-pnext-nightly-275\Level1\src\Starcounter\Apps\GatewayHandlers.cs:line 784
ErrorCode=4174
HResult=-2146233088
HelpLink=https://github.com/Starcounter/Starcounter/wiki/SCERR4174
```
##### Starcounter version

 2.3.0.4048
##### Involved apps' version
- StarcounterPrefabs/Launcher on master - 37b9651
- StarcounterPrefabs/SignIn on develop - cc44730
- StarcounterPrefabs/UserAdmin on master - 2e874bc

**Comment by warpech** [comment]
@diliger could you please investigate this?

**Comment by mmnosek** [comment]
Fixed in #51 Thanks @diliger !

**Comment by warpech** [comment]
Thank you guys!
